init-tools.cmd

```diff
@@ -81,6 +81,36 @@ if not [%INIT_TOOLS_ERRORLEVEL%]==[0] (
   goto :error
 )
 
+:: Restore a custom RoslynToolset since we can't trivially update the BuildTools dependency in CoreRT
+echo Configurating RoslynToolset...
```

Comment:
This is basically a clone of the logic in `"%BUILD_TOOLS_PATH%\init-tools.cmd"`, but restoring the required roslyn toolset version.

Reply:
need to the corresponding logic to `init-tools.sh` as well...

Reply:
Done.